apps/web/src/services/iam/components/UserManagementStatusModal.vue

```diff
@@ -59,13 +66,46 @@ const state = reactive({
     filteredServices: undefined,
     filteredItems: [],
     filteredUniqueItems: [],
+    selectedOnlyWorkspaceUsers,
 });
 
+watch(
+    [
+        () => workspaceUserList.value,
+        () => userGroupListData.value,
+        () => state.filteredUniqueItems,
+        () => serviceListQuery.data.value,
+    ],
+    ([workspaceUsers, userGroups, filteredItems, serviceList]) => {
+        if (!workspaceUsers || !userGroups || !filteredItems?.length || !serviceList) return;
+
+        selectedOnlyWorkspaceUsers.value = workspaceUsers
+            .filter((user) => !user.role_binding_info?.workspace_group_id
+                && filteredItems.map((item: any) => item.user_id).includes(user.user_id))
+            .map((user) => {
+                const userServiceNames: string[] = [];
+                Object.values(serviceList).forEach((service: any) => {
+                    if (service?.members?.USER?.includes(user.user_id)) {
+                        userServiceNames.push(service.name);
+                    }
+                });
+                return {
+                    ...user,
+                    user_group: userGroups
+                        .filter((group) => group.users?.includes(user.user_id ?? ''))
+                        .map((group) => group.name),
+                    service: userServiceNames,
+                };
+            });
+    },
+    { immediate: true, deep: true },
+);
+
```

Comment:
Suggestion: how about using computed~?

Reply:
I think, `state.filteredUniqueItems` can be made by `compute` too.

Reply:
Agree!!